.github/workflows/helm.yaml

```diff
@@ -0,0 +1,33 @@
+name: helm sanity
```

Comment:
I have no idea what this file means, hence I approve it

Reply:
this is the github action